src/guides/v2.3/frontend-dev-guide/css-topics/css-preprocess.md

```diff
@@ -101,7 +101,10 @@ The tool pre-processes (including compilation) and publishes the static view fil
 {:.bs-callout-info}
 Manual static content deployment is not required in "default" and "developer" modes. If you still want to deploy in these modes, use the -f option: `bin/magento setup:static-content:deploy -f`. Read more about the command in the [Deploy static view files]({{ page.baseurl }}/config-guide/cli/config-cli-subcommands-static-view.html) section.
 
-All errors occurring during `.less` files compilation are handled by the [`oyejorge/less.php`](https://github.com/oyejorge/less.php) third party library.
+All errors occurring during `.less` files compilation are handled by the [`wikimedia/less.php`]([LESS PHP library]) third party library.
+
+{:.bs-callout-info}
+Since CE 2.3.4 package oyegorge/less.php has already replaced by wikimedia/less.php
```

Comment:
If CE = Community Edition , please replace with Magento Open Source (It is not called Community Edition anymore) - also what about the Commerce version?
Also surround the file names with backquotes